routes/net-state-files.go

```diff
@@ -0,0 +1,79 @@
+package routes
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/julienschmidt/httprouter"
+
+	"github.com/storj/storage/boltdb"
+)
+
+var (
+	ErrReadReq = errors.New("error reading request body")
+)
+
+type File struct {
+	DB *boltdb.Client
+}
+
+type Message struct {
+	Value string `json:"value"`
+}
+
+func NewFile(db *boltdb.Client) *File {
+	return &File{DB: db}
+}
+
+func (f *File) Put(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+	givenPath := ps.ByName("path")
+
+	var msg Message
+	decoder := json.NewDecoder(r.Body)
+	err := decoder.Decode(&msg)
+	if err != nil {
+		log.Printf("err decoding response: %v", err)
+		return
```

Comment:
dumb nitpick, but technically this means we won't respond to the request at all. I think we need to do `http.Error(w, "bad request!", http.StatusBadRequest)` or something similar to convey the error to the client

Reply:
I think it would also be useful for these errors if we incremented a metric

Reply:
is there a specific variable metric I should make and increment now @coyle or maybe just make a comment for the future?